components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/ConfigurationManagerImpl.java

```diff
@@ -531,6 +531,39 @@ private void validateResourceCreateRequest(String resourceTypeName, ResourceAdd
         }
     }
 
+    private void validateResourceCreateRequest(String resourceTypeName, Resource resource)
```

Comment:
Add a method description on what validations are done inside the method.

Reply:
Fixed in d8a058a